CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+
+# How you can contribute
+
+Thanks for your interest in contributing to VISCtemplates! There are several ways you can contribute to this package:
+
+* **Propose an idea.** Do you have an idea for a new VISCtemplates feature? Take a look at the issue list first to see if it isn't included or suggested yet. If not, suggest your idea as an issue on GitHub.
+
+* **Report a bug.** Report a bug as an issue on GitHub so we can fix it. When filing an issue, the most important thing is to include a minimal reproducible example so that we can quickly verify the problem, and then figure out how to fix it. There are three things you need to include to make your example reproducible: required packages, data, code.
+
+    1. Packages should loaded at the top of the script.
+    2. The easiest way to include data is to use `dput()` to generate the R code to recreate it. But even better is if you can create a data.frame() with just a handful of rows and columns that still illustrates the problem.
+    3. Spend a little bit of time ensuring that your code is easy for others to read: (1) Make sure you've used spaces and your variable names are concise, but informative. (2) Use comments to indicate where your problem lies. (3) Do your best to remove everything that isn't related to the problem. The shorter your code is, the easier it is to understand.
+<br><br>
+
```

Comment:
This is a terrific explanation of how to create a **MRE**! Wondering if the concept of a bug report should draw from the "show don't tell" principle? The components are: 
1. What you did. 
2. What happened. 
3. What you expected to happen.

I've personally found that while a good MRE is invaluable to getting a solution off the ground, 2 & 3 provide context for deeper discussion.

Reply:
I stole it from the ggplot2 contributing.md! 

Do you think I should link to a longer explanation of how to create a MRE or a something about the "show don't tell" principle?

Reply:
There is a nice write up of "show don't tell" within the framework of bug reporting on the RStudio blog: https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports. Maybe just link to this for now?